src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,204 @@
+      justify-content: flex-start;
+
+      &:before,
+      &:after {
+        flex-basis: 0;
+      }
+    }
+  }
+
+  .govuk-pagination__item {
+    @include govuk-font(19);
+    display: none; // Hide items on small screens except the prev/next items, non-link items and the first and last items
+    position: relative;
+    margin-bottom: govuk-spacing(4);
+    float: left; // Float is ignored if flex is active
+    order: 2;
+
+    // Hide items on small screens
+    @include govuk-media-query($from: tablet) {
+      display: block;
+    }
+
+    &:hover {
+      background-color: govuk-colour("light-grey", $legacy: "grey-4");
+    }
+  }
+
+  .govuk-pagination__icon {
+    // Set size using rems to make the icon scale with text if user resizes text in their browser
+    width: govuk-px-to-rem(15px);
+    height: govuk-px-to-rem(13px);
+    margin-top: govuk-spacing(1);
+    fill: $govuk-secondary-text-colour;
+  }
+
+  .govuk-pagination__icon--prev {
+    margin-right: govuk-spacing(3);
+  }
+
+  .govuk-pagination__icon--next {
+    margin-left: govuk-spacing(3);
+  }
+
+  // Only show previous, next, first, last and non-link items on mobile
+  .govuk-pagination__item--current,
+  .govuk-pagination__item--ellipses,
+  .govuk-pagination__item--prev,
+  .govuk-pagination__item--next,
+  .govuk-pagination__item:nth-child(2),
+  .govuk-pagination__item:nth-last-child(2) {
+    display: block;
+  }
+
+  .govuk-pagination__item,
+  .govuk-pagination__item--current,
+  .govuk-pagination__item--ellipses {
+    box-sizing: border-box;
+    min-width: govuk-spacing(8);
+    min-height: govuk-spacing(8);
+    padding: govuk-spacing(2) govuk-spacing(3);
+  }
+
+  .govuk-pagination__item--current,
+  .govuk-pagination__item--ellipses {
+    font-weight: bold;
+    text-align: center;
+  }
+
+  .govuk-pagination__item--ellipses {
+    color: $govuk-secondary-text-colour;
+  }
+
+  .govuk-pagination__link {
+    display: block;
+    min-width: govuk-spacing(3);
+    text-align: center;
+
+    &:after {
+      content: "";
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+    }
+
+    // Add link hover decoration to prev/next text if no label present on prev/next only mode
+    // We do this so that we have a hover state in all possible instances
+    &:hover,
+    &:active {
+      .govuk-pagination__link-title--decorated {
+        @include govuk-link-decoration;
+      }
+
+      .govuk-pagination__link-label,
+      .govuk-pagination__link-title--decorated {
+        @include govuk-link-hover-decoration;
+      }
+    }
+
+    &:focus {
+      .govuk-pagination__icon {
+        fill: $govuk-focus-text-colour;
+      }
+
+      .govuk-pagination__link-label {
+        text-decoration: none;
+      }
+    }
+  }
+
+  .govuk-pagination__link-label {
+    // Don't set the font via govuk font as currently this will mean the font size is 14px on mobile
+    // which is an inaccessible font size
```

Comment:
This to me is more credence to fix the issues with typography more urgently. Whilst the issues here are issues, not passing the font is the lesser evil compared to shipping an inaccessible font size.

Something I can try here is still pass the font so that it adds the helper but maintain the font-size declaration on line 147. I'm not 100% sure that's going to work and it's messy but could be worth a shot?